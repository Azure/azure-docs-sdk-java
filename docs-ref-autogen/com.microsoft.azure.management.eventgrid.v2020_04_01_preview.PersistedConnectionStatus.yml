### YamlMime:ManagedReference
items:
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
  id: PersistedConnectionStatus
  artifact: com.microsoft.azure.eventgrid.v2020_04_01_preview:azure-mgmt-eventgrid:1.0.0-beta-3
  parent: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
  children:
  - com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.APPROVED
  - com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.DISCONNECTED
  - com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.PENDING
  - com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.PersistedConnectionStatus()
  - com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.REJECTED
  - com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.fromString(java.lang.String)
  - com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.values()
  langs:
  - java
  name: PersistedConnectionStatus
  nameWithType: PersistedConnectionStatus
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
  type: Class
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
  summary: Defines values for PersistedConnectionStatus.
  syntax:
    content: public final class PersistedConnectionStatus extends ExpandableStringEnum<PersistedConnectionStatus>
  inheritance:
  - java.lang.Object
  - com.microsoft.rest.ExpandableStringEnum
  inheritedMembers:
  - com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)
  - com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)
  - com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)
  - com.microsoft.rest.ExpandableStringEnum.hashCode()
  - com.microsoft.rest.ExpandableStringEnum.toString()
  - com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)
  - java.lang.Object.clone()
  - java.lang.Object.finalize()
  - java.lang.Object.getClass()
  - java.lang.Object.notify()
  - java.lang.Object.notifyAll()
  - java.lang.Object.wait()
  - java.lang.Object.wait(long)
  - java.lang.Object.wait(long,int)
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.APPROVED
  id: APPROVED
  artifact: com.microsoft.azure.eventgrid.v2020_04_01_preview:azure-mgmt-eventgrid:1.0.0-beta-3
  parent: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
  langs:
  - java
  name: APPROVED
  nameWithType: PersistedConnectionStatus.APPROVED
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.APPROVED
  type: Field
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
  summary: Static value Approved for PersistedConnectionStatus.
  syntax:
    content: public static final PersistedConnectionStatus APPROVED
    return:
      type: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.DISCONNECTED
  id: DISCONNECTED
  artifact: com.microsoft.azure.eventgrid.v2020_04_01_preview:azure-mgmt-eventgrid:1.0.0-beta-3
  parent: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
  langs:
  - java
  name: DISCONNECTED
  nameWithType: PersistedConnectionStatus.DISCONNECTED
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.DISCONNECTED
  type: Field
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
  summary: Static value Disconnected for PersistedConnectionStatus.
  syntax:
    content: public static final PersistedConnectionStatus DISCONNECTED
    return:
      type: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.PENDING
  id: PENDING
  artifact: com.microsoft.azure.eventgrid.v2020_04_01_preview:azure-mgmt-eventgrid:1.0.0-beta-3
  parent: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
  langs:
  - java
  name: PENDING
  nameWithType: PersistedConnectionStatus.PENDING
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.PENDING
  type: Field
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
  summary: Static value Pending for PersistedConnectionStatus.
  syntax:
    content: public static final PersistedConnectionStatus PENDING
    return:
      type: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.PersistedConnectionStatus()
  id: PersistedConnectionStatus()
  artifact: com.microsoft.azure.eventgrid.v2020_04_01_preview:azure-mgmt-eventgrid:1.0.0-beta-3
  parent: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
  langs:
  - java
  name: PersistedConnectionStatus()
  nameWithType: PersistedConnectionStatus.PersistedConnectionStatus()
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.PersistedConnectionStatus()
  overload: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.PersistedConnectionStatus*
  type: Constructor
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
  syntax:
    content: public PersistedConnectionStatus()
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.REJECTED
  id: REJECTED
  artifact: com.microsoft.azure.eventgrid.v2020_04_01_preview:azure-mgmt-eventgrid:1.0.0-beta-3
  parent: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
  langs:
  - java
  name: REJECTED
  nameWithType: PersistedConnectionStatus.REJECTED
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.REJECTED
  type: Field
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
  summary: Static value Rejected for PersistedConnectionStatus.
  syntax:
    content: public static final PersistedConnectionStatus REJECTED
    return:
      type: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.fromString(java.lang.String)
  id: fromString(java.lang.String)
  artifact: com.microsoft.azure.eventgrid.v2020_04_01_preview:azure-mgmt-eventgrid:1.0.0-beta-3
  parent: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
  langs:
  - java
  name: fromString(String name)
  nameWithType: PersistedConnectionStatus.fromString(String name)
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.fromString(String name)
  overload: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.fromString*
  type: Method
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
  summary: Creates or finds a PersistedConnectionStatus from its string representation.
  syntax:
    content: public static PersistedConnectionStatus fromString(String name)
    parameters:
    - id: name
      type: java.lang.String
      description: a name to look for
    return:
      type: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
      description: the corresponding PersistedConnectionStatus
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.values()
  id: values()
  artifact: com.microsoft.azure.eventgrid.v2020_04_01_preview:azure-mgmt-eventgrid:1.0.0-beta-3
  parent: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
  langs:
  - java
  name: values()
  nameWithType: PersistedConnectionStatus.values()
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.values()
  overload: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.values*
  type: Method
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
  syntax:
    content: public static Collection<PersistedConnectionStatus> values()
    return:
      type: java.util.Collection<com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus>
      description: known PersistedConnectionStatus values
references:
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.PersistedConnectionStatus*
  name: PersistedConnectionStatus
  nameWithType: PersistedConnectionStatus.PersistedConnectionStatus
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.PersistedConnectionStatus
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
- uid: java.lang.String
  spec.java:
  - uid: java.lang.String
    name: String
    fullName: java.lang.String
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.fromString*
  name: fromString
  nameWithType: PersistedConnectionStatus.fromString
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.fromString
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
- uid: java.util.Collection<com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus>
  spec.java:
  - uid: java.util.Collection
    name: Collection
    fullName: java.util.Collection
  - name: <
    fullName: <
  - uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
    name: PersistedConnectionStatus
    fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus
  - name: '>'
    fullName: '>'
- uid: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.values*
  name: values
  nameWithType: PersistedConnectionStatus.values
  fullName: com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.values
  package: com.microsoft.azure.management.eventgrid.v2020_04_01_preview
- uid: com.microsoft.rest.ExpandableStringEnum<com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus>
  name: ExpandableStringEnum<PersistedConnectionStatus>
  nameWithType: ExpandableStringEnum<PersistedConnectionStatus>
  fullName: com.microsoft.rest.ExpandableStringEnum<com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus>
- uid: java.lang.Object.notify()
  name: Object.notify()
  nameWithType: Object.notify()
  fullName: java.lang.Object.notify()
- uid: java.lang.Object.wait()
  name: Object.wait()
  nameWithType: Object.wait()
  fullName: java.lang.Object.wait()
- uid: java.lang.Object.finalize()
  name: Object.finalize()
  nameWithType: Object.finalize()
  fullName: java.lang.Object.finalize()
- uid: java.lang.Object.notifyAll()
  name: Object.notifyAll()
  nameWithType: Object.notifyAll()
  fullName: java.lang.Object.notifyAll()
- uid: java.lang.Object.clone()
  name: Object.clone()
  nameWithType: Object.clone()
  fullName: java.lang.Object.clone()
- uid: com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)
  name: ExpandableStringEnum.withNameValue(String,T,Class<T>)
  nameWithType: ExpandableStringEnum.withNameValue(String,T,Class<T>)
  fullName: com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)
- uid: com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)
  name: ExpandableStringEnum.<T>fromString(String,Class<T>)
  nameWithType: ExpandableStringEnum.<T>fromString(String,Class<T>)
  fullName: com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)
- uid: com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)
  name: ExpandableStringEnum.<T>values(Class<T>)
  nameWithType: ExpandableStringEnum.<T>values(Class<T>)
  fullName: com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)
- uid: com.microsoft.rest.ExpandableStringEnum.toString()
  name: ExpandableStringEnum.toString()
  nameWithType: ExpandableStringEnum.toString()
  fullName: com.microsoft.rest.ExpandableStringEnum.toString()
- uid: com.microsoft.rest.ExpandableStringEnum.hashCode()
  name: ExpandableStringEnum.hashCode()
  nameWithType: ExpandableStringEnum.hashCode()
  fullName: com.microsoft.rest.ExpandableStringEnum.hashCode()
- uid: java.lang.Object.getClass()
  name: Object.getClass()
  nameWithType: Object.getClass()
  fullName: java.lang.Object.getClass()
- uid: java.lang.Object.wait(long)
  name: Object.wait(long)
  nameWithType: Object.wait(long)
  fullName: java.lang.Object.wait(long)
- uid: com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)
  name: ExpandableStringEnum.equals(Object)
  nameWithType: ExpandableStringEnum.equals(Object)
  fullName: com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)
- uid: java.lang.Object.wait(long,int)
  name: Object.wait(long,int)
  nameWithType: Object.wait(long,int)
  fullName: java.lang.Object.wait(long,int)
- uid: java.util.Collection
  name: Collection
  nameWithType: Collection
  fullName: java.util.Collection
- uid: com.microsoft.rest.ExpandableStringEnum
  name: ExpandableStringEnum
  nameWithType: ExpandableStringEnum
  fullName: com.microsoft.rest.ExpandableStringEnum
- uid: com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class
  name: ExpandableStringEnum.withNameValue(String,T,Class
  nameWithType: ExpandableStringEnum.withNameValue(String,T,Class
  fullName: com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class
- uid: T>)
  name: T>)
  nameWithType: T>)
  fullName: T>)
- uid: com.microsoft.rest.ExpandableStringEnum.
  name: ExpandableStringEnum.
  nameWithType: ExpandableStringEnum.
  fullName: com.microsoft.rest.ExpandableStringEnum.
- uid: T>fromString(java.lang.String,java.lang.Class
  name: T>fromString(String,Class
  nameWithType: T>fromString(String,Class
  fullName: T>fromString(java.lang.String,java.lang.Class
- uid: T>values(java.lang.Class
  name: T>values(Class
  nameWithType: T>values(Class
  fullName: T>values(java.lang.Class
