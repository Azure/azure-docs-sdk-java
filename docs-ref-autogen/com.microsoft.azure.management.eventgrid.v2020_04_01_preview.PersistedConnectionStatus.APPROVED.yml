### YamlMime:JavaMember
uid: "com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.APPROVED*"
fullName: "com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PersistedConnectionStatus.APPROVED"
members:
- uid: "com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.APPROVED"
  fullName: "com.microsoft.azure.management.eventgrid.v2020_04_01_preview.PersistedConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PersistedConnectionStatus.APPROVED"
  summary: "Static value Approved for PersistedConnectionStatus."
  syntax: "public static final PersistedConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.microsoft.azure.management.eventgrid.v2020_04_01_preview"
artifact: com.microsoft.azure.eventgrid.v2020_04_01_preview:azure-mgmt-eventgrid:1.0.0-beta-3
